submission/stats/src/main/kotlin/uk/ac/ebi/stats/service/SubmissionStatsService.kt

```diff
@@ -37,7 +37,12 @@ open class SubmissionStatsService(
     }
 
     @Transactional
-    open fun saveAll(stats: List<SubmissionStat>): List<SubmissionStat> = stats.map { save(it) }
+    open fun saveAll(stats: List<SubmissionStat>): List<SubmissionStat> = stats.map(::save)
+
+    @Transactional
+    open fun incrementAll(
+        stats: List<SubmissionStat>
+    ): List<SubmissionStat> = stats.map(::increment).map(::save)
```

Comment:
I think rather here we want to map and then call `saveAll`

Reply:
Done